.circleci/config.yml

```diff
@@ -140,6 +140,36 @@ jobs:
       - run: composer functional
       - run: composer integration
 
+
+  # PHP 7.4 test with HIGHEST dependencies:
+  #   Determines whether a newer version of a dependency has broken Drush.
+  test_74_drupal91:
+    <<: *defaults
+    docker:
+      - image: wodby/php:7.4-dev
+        environment:
+          - MYSQL_HOST=127.0.0.1
+          - UNISH_DB_URL=mysql://root:@127.0.0.1
+      - image: circleci/mysql:5.7.24
+    steps:
+      - checkout
+      - run: sudo composer self-update --preview
+      - run: cp .docker/zz-php.ini /usr/local/etc/php/conf.d/
+      # We test with 9.0.0 so that \Unish\SecurityUpdatesTest can find an insecure drupal/core.
+      # Remove that assertion once testing with 9.0.0 becomes stale.
+      - run: composer require --dev drupal/core-recommended:~9.1.0 --no-update
+      # Replace alinks with semver for D9.
+      - run: composer remove --dev drupal/alinks --no-update
+      - run: composer require --dev drupal/semver_example:2.2.0 --no-update
+      - run: composer config platform.php 7.3.0
```

Comment:
Should be removed or changed to 7.4

Reply:
I bet it still required as https://github.com/drush-ops/drush/blob/10.x/composer.json#L83-L84 sticking it on 7.1.3